tasks/local.rake

```diff
@@ -1,18 +1,19 @@
+require_relative "application_generator"
+
 desc 'Runs a command agains the local sample application'
 task :local do
-  Rake::Task['setup'].invoke(false,
-                             'development',
-                             'rails_template_with_data')
+  ActiveAdmin::ApplicationGenerator.new(
+    rails_env: 'development',
+    template: 'rails_template_with_data'
+  ).generate
 
   app_folder = ".test-rails-apps/rails-#{Rails::VERSION::STRING}"
```

Comment:
Since we also have this in the ApplicationGenerator class, I wonder if we can avoid that duplication. If you don't want to store access to an instance of the generator, perhaps an idea could be that the `generate` method returns the `app_dir` value? It doesn't seem far fetched since that's really what the method deals a lot with and would signify what's there. Or perhaps something else?

Reply:
Sounds like a good idea to me! I'll add it!

Reply:
I pushed https://github.com/activeadmin/activeadmin/pull/5530/commits/78e2aca1e5e2d4feb944414cb84fca68fabf0454!